src/plans/proximal_gradient_plan.jl

```diff
@@ -14,7 +14,7 @@ as well as ``\operatorname{grad} g`` and ``\operatorname{prox}_{λ} h``.
 # Fields
 
 * `cost`: the overall cost ``f = g + h``
-* `cost_g`: the smooth cost component ``g``
+* `cost_smooth`: the smooth cost component ``g``
 * `gradient_g!!`: the gradient ``\operatorname{grad} g``
 * `proximal_map_h!!`: the proximal map ``\operatorname{prox}_{λ} h``
```

Comment:
```suggestion
* `cost_smooth`: the smooth cost summand ``g``
* `gradient_smooth!!`: the gradient of the smooth cost summand``\operatorname{grad} g``
* `proximal_map_nonsmooth!!`: the proximal map ``\operatorname{prox}_{λ} h`` of the non smooth cost summand ``h``.
```

If we rename one to smooth, maybe we should consistently speak about smooth and non smooth summands (or components) of the cost?